libbeat/otelbeat/beatconverter/beatconverter_test.go

```diff
@@ -43,10 +43,18 @@ exporters:
       max_retries: 3
     user: elastic
     timeout: 1m30s
-    batcher:
+    force_attempt_http2: false
```

Comment:
I think this is probably handled by https://github.com/elastic/elastic-agent/pull/9257 with the change in how we use Beats auth extension to just use the elastic-agent-libs HTTPTransport directly which set this. We'll want this in the future but I think this has no effect anymore so we should omit it.

CC @khushijain21 this will either need to be adjusted here or in https://github.com/elastic/beats/pull/46428 depending on the merge order.